reference/batch-workflow-syntax.md

```diff
@@ -189,6 +204,171 @@ cache:
 
 **Expression contexts:** [`flow` context](batch-contexts.md#flow-context), [`params` context](batch-contexts.md#params-context), [`env` context](batch-contexts.md#env-context), [`tags` context](batch-contexts.md#tags-context), [`volumes` context](batch-contexts.md#volumes-context), [`images` context](batch-contexts.md#images-context).
 
+## `images`
+
+A mapping of image definitions used by the _batch_ workflow.
+
+`neuro-flow build <image-id>` creates an image from the passed `Dockerfile` and uploads it to the Neu.ro Registry. The `${{ images.img_id.ref }}` expression can be used for pointing the image from a [`tasks.image`](batch-workflow-syntax.md#tasks-image).
```

Comment:
Sorry, this is not correct. This is how it works for live mode, in batch mode building happens automatically during bake progress.

Reply:
Updated in the most recent commit.